core/domain/story_services.py

```diff
@@ -212,6 +221,20 @@ def apply_change_list(story_id, change_list):
         raise
 
 
+def check_url_fragment_uniqueness(url_fragment):
```

Comment:
This should be framed more as "does_story_exist_with_url_fragment" instead -- "uniqueness" applies when you're looking at a list of stuff and want to check that each element appears once.